tests/waveplate_stub_test.py

```diff
@@ -0,0 +1,108 @@
+from pnpq.devices.waveplate_stub import WaveplateStub
+from pnpq.errors import (
+    DeviceDisconnectedError,
+    WaveplateInvalidDegreeError,
+    WaveplateInvalidMotorChannelError,
+    WaveplateInvalidStepsError,
+)
+import pytest
+
+def test_access_without_connection():
+    wp = WaveplateStub()
+    with pytest.raises(DeviceDisconnectedError):
+        wp.identify()
+
+def test_connect():
```

Comment:
This doesn't test `connect`, it tests `identify`.

Reply:
In the stub device `connect` do not fail, but the goal of the test is to check whether the device works after calling `connect` (and test whether requested are rejected if `connect` is not called). `identify` happens to be the simplest function to test this, as in the stub it doesn't do anything (but fails if the device is not connected)

Reply:
I modified it so that it says `test_identify`